dydx/dydxPresenters/dydxPresenters/_v4/Receipt/dydxTradeReceiptPresenter.swift

```diff
@@ -43,17 +43,16 @@ final class dydxTradeReceiptPresenter: dydxReceiptPresenter {
                 AbacusStateManager.shared.state.selectedSubaccountPositions,
                 AbacusStateManager.shared.state.marketMap)
             .sink { [weak self] input, positions, marketMap in
-                if let tradeSummary = input.summary,
-                    let marketId = input.marketId,
-                    let market = marketMap[marketId],
-                    let position = positions.first(where: { $0.id == marketId }) {
-                    self?.updateExpectedPrice(tradeSummary: tradeSummary, market: market)
-                    self?.updateLiquidationPrice(position: position, market: market)
-                    self?.updatePositionMargin(position: position)
-                    self?.updatePositionLeverage(position: position)
-                    self?.updateTradingFee(tradeSummary: tradeSummary)
-                    self?.updateTradingRewards(tradeSummary: tradeSummary)
-                }
+                let tradeSummary = input.summary
+                let marketId = input.marketId
+                let market = marketMap[marketId ?? ""]
+                let position = positions.first(where: { $0.id == marketId })
+                self?.updateExpectedPrice(tradeSummary: tradeSummary, market: market)
+                self?.updateLiquidationPrice(position: position, market: market)
+                self?.updatePositionMargin(position: position)
+                self?.updatePositionLeverage(position: position)
+                self?.updateTradingFee(tradeSummary: tradeSummary)
+                self?.updateTradingRewards(tradeSummary: tradeSummary)
```

Comment:
this change ensures the initial (empty) state loads

despite null values, we still want to display the receipt line titles